### ISSUE TITLE
Option to choose movie folder & file naming in a language and / or encoding my keyboard supports 

### ISSUE THREAD

**Issue by tvebax** [Radarr/Radarr#4046]
**Is your feature request related to a problem? Please describe.**
Related to:
https://github.com/Radarr/Radarr/issues/4045

Lately Radarr has started naming movie files and folders in the movie's original language. You could say language now disregards the user locale, in favor of the movie locale. Not a great experience.

Examples:
הערת שוליים
جدایی نادر از سیمین
درباره الی‎‎
おくりびと
万引き家族
기생충
Zimna wojna
Vals Im Bashir
Tro, håb og kærlighed
Tarde para la ira
Sprängaren

While I do watch movies in these languages, I do not understand them - and several I have no idea how to type on my keyboard. Having folders named things like "万引き家族 (2018)" means I cannot cd into them (my keyboard is standard english, no way to type chinese characters..) - plus the name makes 0 sense to me. 

This is my Movie Folder Format setting:
`{Movie CleanTitle} ({Release Year})`

I am not sure what CleanTitle cleans, but it definitely is not character set.

**Describe the solution you'd like**
I would like an option to configure a "Movie Folder Format" and "Standard Movie Format" in a language I understand and in a character encoding my keyboard and system supports.

Ideally I would love to be able to specify "English" as language and some ISO code as format to avoid UTF characters all together.

Goal: Have all movie folder and file names in plain english, with no special characters except for the () around release year.

**Comment by Qstick** [comment]
Believe this is a duplicate request of #3154

**Comment by tvebax** [comment]
Not exactly. Minimum viable product for the request here, would be an option to sest Radarr language globally to english and thereby disable any multilingual features.

Also:
I was in doubt whether to lodge this as a feature request or as a bug / regression from implementing multilingual features. This is not (!) a request to enhance multilingual features. This is a request to avoid messing up existing english only installs (regression).

**Comment by fryfrog** [comment]
I think it is close enough to the general "make Radarr better w/ languages" of many other feature requests. But is this a regression? Hasn't it always behaved this way?

**Comment by majora2007** [comment]
Interesting. I used to have this exact issue a while ago where Radarr would create Japanese or Russian character folders, but it hasn't in quite some time. This should definitely be a bug imo.

**Comment by tvebax** [comment]
No - things used to generally be in english all over.

I do applaud that you are implementing multilingual features to support folks all over the world. But the larger user base understand english. These installs are now messed up.

Currently I am on my 3rd round of several hours tedious manual corrections to title names over the last few months :(
Neither my keyboard or my setup supports chinese / arabic / etc, so aside from being messy and incomprehensible.. The new forced multilingual "features" break other parts of my system.

Super painful and definitely regression.

I have had Radarr running for years, it never broke my setup before, never needed manual fixing before.
Being selfish, I really wish you would just stop "make Radarr better w/ languages". But perhaps at least offer some option to keep a stable setup for the major part of users who don't care about multilingual?

Actually. The way things work currently, I doubt there is a single user in the world which fits the usecase.. Radarr is being developed so you need to be able to understand and handle paths / file names in every language you have a movie in. I don't know a single person who speaks the 20 languages or more I have movies in. Let alone have keyboards that can write in them...

**Comment by fryfrog** [comment]
Do you have static paths on or off? I *think* you might be able to "solve" it by making sure they're *on*. I believe all that Radarr gets initially is the English data, so the folder is created w/ that. *Afterward*, it gets all the other data and the folder can get renamed if you have dynamic naming enabled.

**Comment by tvebax** [comment]
I am aware "rename folders" can be turned off. This is however a necessary feature.

Without it, folder names end up as "[Movie Title] (0)". The result is Plex finds no match - or worse, the wrong match. Also, once in a while the (english) movie title gets changed along the way which also causes issues with Plex matching, if the folder name does not reflect this. 

Especially the wrong matches are easy to miss, so without the folder renaming it is necessary to manually check every new movie added.

**Comment by fryfrog** [comment]
I can't disagree. You'll just have to make the choice between these two issues. Which bothers you more?

If you feel strongly about this, I'd honestly suggest learning .NET and working on making it happen w/ the developers. Have a look at the age of the other language issues, I think you'll see it just isn't a very high priority.

**Comment by tvebax** [comment]
To be honest, I wish the multilingual stuff would be much *less* of a priority.
Radarr worked beautifully in plain english until multilanguage became a thing (priority?).
Sonarr doesn't have it either, works perfectly.

I am not native english speaking myself and did years working on multilingual apps. Multilingual is surprisingly complex to implement. I really wish Radarr would stay away. Not a good project to pick, unless you're sure you got piles of resources to pour into it.

The complexity of this initiative and amount of related bugs in the queue, could perhaps warrant multilingual work being isolated in it's own branch / fork? Like the v3 aphrodite?
Currently all 3 branches are broken w regards to language.

Regarding development - I would, if it was anything but .Net. Fits nowhere in the rest of my life / work. So, I will stay grateful for the work others do on this one.

**Comment by sam272** [comment]
Seconding this as a bug/regression. Until very recently for me (this week) the English titles were being used to name titles when "Any" language was set in profile. Suddenly all those movies are being renamed to their original language titles which isn't very useful when you can't read that language. The "Any" profile _is_ very useful because it lets you download files that are named with the original language title - but not with this behavior. Would appreciate a switch back or update that allows the choice of what language to name titles. Thanks!

**Comment by ngthunter** [comment]
I have the the same problem the last days.
There are many non English movies in my library (especially korean, japanese etc) and every single one of them has it's original title now.

And the worst thing is I can't even search for them using the english title in radarr, they just don't show up

**Comment by ngthunter** [comment]
Any update? Still having the same problems with non english movies. I have to keep english as a language in every profile so I miss many movies
